src/components/util/util.js

```diff
@@ -199,3 +200,10 @@ export const getRequestUrl = (barcode, userId) => {
       userId,
     })}`;
 };
+
+export const isPatronUser = (user) => user?.type === USER_TYPES.PATRON;
+export const isDcbUser = (user) => user?.type === USER_TYPES.DCB;
+
+export const isAffiliationsEnabled = (user) => {
+  return !isPatronUser(user) && !isDcbUser(user);
+};
```

Comment:
Please add tests. I know these functions are dead-simple. At least the tests will be dead-simple too.

Reply:
Sure, I've added.